CHANGELOG.md

```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [0.7.0] - 2021-07-01
+
+### Compatibility Changes:
+- Visual Studio 2017 is no longer supported #1251
+- 32bit Windows is no longer supported #1251
+- CUDA 11.3 is now supported #1295
+- clang < 9 is no longer supported as CUDA compiler #1300
+- clang 11 is now supported #1310
+
+### Bug Fixes:
+- fixed static shared memory allocation for OpenMP 5 back-end #1254
+- fixed static shared memory allocation for CPU back-ends #1258
+- fixed `ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED` being checked without being defined #1259
+- fixed `BlockSharedMemStMemberImpl::getVarPtr` for last variable #1280
+- fixed static shared memory alignment #1282
+- fixed `BlockSharedMemStMemberImpl::getVarPtr` for last variable #1280
+- fixed static shared memory alignment #1282
+- fixed inverted assertion in `DeclareSharedVar` #1303
+- fixed OpenMP scheduler support #1307 #1309 #1316
+
```

Comment:
The PR #1129 seems to be missing. Please add something like:
`forward ALPAKA_CXX_STANDARD to nvcc on Windows: #1129`

Reply:
Issues will not be part of the CHANGELOG, only PRs which solves issues. The PR to solve #1129 should be #1130

Reply:
Right! So it should be #1130. Thx!

Reply:
#1130 is already part of 0.6.0 (according to the assigned milestone). Did this change and it is actually part of 0.7.0?

Reply:
Then the issue #1129 was wrongly assigned to 0.7.0, I changed it to 0.6.0.